packages/utils/src/validation/validatorFunctions.js

```diff
@@ -87,6 +87,14 @@ export const isPlainObject = value => {
   }
 };
 
+export const assertAllValuesAreNumbers = object => {
```

Comment:
```suggestion
export const allValuesAreNumbers= object => {
```

`assert` is a more "correct" prefix in theory, but omitting it makes config declarations very readable. I think I prefer omitting it, but the actual reason I suggest this change is for consistency with existing validator function names.

Reply:
That makes sense.